src/main/webapp/app/complaints/list-of-complaints/list-of-complaints.component.ts

```diff
@@ -30,7 +30,7 @@ export class ListOfComplaintsComponent implements OnInit {
     private exerciseId: number;
     private tutorId: number;
     private examId?: number;
-    private correctionRound?: number;
+    correctionRound?: number;
```

Comment:
```suggestion
    private correctionRound?: number;
```
I don’t think this is accessed directly in the tests any more and can be set to `private`.

Reply:
It is, `list-of-complaints.component.spec.ts:325`